backend-rust/src/indexer.rs

```diff
@@ -3223,7 +3219,7 @@ async fn process_cis2_token_event(
                 token_address,
                 contract_index,
                 contract_sub_index,
-                metadata_url.as_str(),
+                metadata_url.url(),
```

Comment:
I would rather do the replacement on the indexing. I am not even sure this will work

Reply:
It will be done as part of indexing. The replacement is done in the conversion code, which is used by the indexer